src/main/java/com/spotify/docker/client/messages/NetworkSettings.java

```diff
@@ -70,6 +72,46 @@
   @JsonProperty("Networks")
   public abstract ImmutableMap<String, AttachedNetwork> networks();
 
+  @Nullable
+  @JsonProperty("EndpointID")
+  public abstract String endpointId();
+
+  @Nullable
+  @JsonProperty("SandboxID")
+  public abstract String sandboxId();
+
+  @Nullable
+  @JsonProperty("SandboxKey")
+  public abstract String sandboxKey();
+
+  @Nullable
+  @JsonProperty("HairpinMode")
+  public abstract Boolean hairpinMode();
+
+  @Nullable
+  @JsonProperty("LinkLocalIPv6Address")
+  public abstract String linkLocalIpv6Address();
+
+  @Nullable
+  @JsonProperty("LinkLocalIPv6PrefixLen")
+  public abstract Integer linkLocalIpv6PrefixLen();
+
+  @Nullable
+  @JsonProperty("GlobalIPv6Address")
+  @SuppressFBWarnings(value = {"NM_CONFUSING"},
+          justification = "Conforming to docker API")
+  public abstract String globalIpv6Address();
+
+  @Nullable
+  @JsonProperty("GlobalIPv6PrefixLen")
+  @SuppressFBWarnings(value = {"NM_CONFUSING"},
+          justification = "Conforming to docker API")
+  public abstract Integer globalIpv6PrefixLen();
```

Comment:
Why is this camel-casing different from `globalIPv6Address`? We use the latter pattern so it'd be nice to be consistent.